### ISSUE TITLE
npm install fail

### ISSUE THREAD

**Issue by baitouwei** [bobmk2/quihex#3]
I like this tool,but run `npm install -g quihex` fail.
log:

```
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/node/5.8.0/bin/node" "/usr/local/bin/npm" "install" "-g" "quihex"
npm ERR! node v5.8.0
npm ERR! npm  v3.7.3
npm ERR! path /usr/local/lib/node_modules/quihex/build/index.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/quihex/build/index.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/quihex/build/index.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/bai/workplace/hexo_blog/npm-debug.log
```

Test at [here](https://tonicdev.com/npm/quihex) also fail.

**Comment by bobmk2** [comment]
Thank you for your report :bow: 

I've resolved this issue at v0.0.2.

**Comment by baitouwei** [comment]
Cool
